src/misc/person.v1.yaml

```diff
@@ -26,12 +26,37 @@ properties:
                       - Schekman, Randy
                       - VijayRaghavan, Krishnaswamy
                       - Li Wenhui
+            given_names:
+                title: Given names
+                description: |
+                    The given names of the person.
+
+                    For example:
+                      - Randy
+            surname:
+                title: Surname
+                description: |
+                    The surname of the person.
+
+                    For example:
+                      - Schekman
         required:
           - preferred
           - index
+          - last
```

Comment:
Should this be `surname` now?

Reply:
Also, if making this new field required means backward compatibility issues - is that the right step forward?

Reply:
We don't need this to be required for all people, only those that xPub is pulling in. So, it should remain optional for now. We could make it required and bump the version if we can populate the last name for all people. Leave as optional for now.

Reply:
It's not a breaking change to add a new required field to a response, but does require the data being available everywhere.